crates/bevy_feathers/src/cursor.rs

```diff
@@ -80,27 +88,33 @@ pub(crate) fn update_cursor(
     cursor_query: Query<&EntityCursor, Without<Window>>,
     q_windows: Query<(Entity, Option<&CursorIcon>), With<Window>>,
     r_default_cursor: Res<DefaultCursor>,
+    r_override_cursor: Res<OverrideCursor>,
 ) {
-    let cursor = hover_map
-        .and_then(|hover_map| match hover_map.get(&PointerId::Mouse) {
-            Some(hover_set) => hover_set.keys().find_map(|entity| {
-                cursor_query.get(*entity).ok().or_else(|| {
-                    parent_query
-                        .iter_ancestors(*entity)
-                        .find_map(|e| cursor_query.get(e).ok())
+    let cursor = (**r_override_cursor)
+        .as_ref()
```

Comment:
You should be able to write it simply as `r_override_cursor.as_ref()`  or as `r_override_cursor.0.as_ref()`.  No need to manually deref.